changes/517.added

```diff
@@ -0,0 +1,2 @@
+Added Ansible Lint to the project.
+Added VLAN Group capability to vlan module.
```

Comment:
I don't think we want this changelog line to link back to the issue about ansible-lint. Let's create a separate issue for it and move this line to its own file.